### DISCUSSION TITLE
Making test proposal

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#31] [General]
Hello all!

@hackfisher I can't find where we discussed on making test proposal before making real one. Could you please help me how to do one. I can't seem to do it on snapshot.

I am planning on opening discussion later this week about moving some funds from Darwinia treasury to DC DAO. Hope many will join discussion. And why we can use them what ammount to stake.

I also plan on opening discussion about KTON SubDAO @hackfisher suggested if there is any interest.  

Step by step. 
Long live DC DAO

**Comment by Laki389** [comment]
> @hackfisher I can't find where we discussed on making test proposal before making real one. Could you please help me how to do one. I can't seem to do it on snapshot.

@SasoLithops I think you are talking about this discussion?[https://github.com/orgs/dcdao/discussions/23]( https://github.com/orgs/dcdao/discussions/23)

> Posting a draft of the proposal on dcdao/discussion to test the water before proposal on snapshot is always a good practice.
> 
> Other voters can give feedback to or help improve it, the fiscal assistant members also can also give advice on budget and funding.

I understand @hackfisher  comment that you first open a discussion here on DCDAO Github with a draft version of your proposal to feel the pulse/opinion of other community members and if you see that the proposal would be voted on, then put it officially on Snapshot for voting.

**Reply by SasoLithops** [reply]
Yes Laki to the rescue again🙌. Thanks. Was looking exactlly for that. Now I see he meant proposal not testing voting. 
Well I am going to absolutely like I wrote above post proposal here first so everyone can comment.  
And only after we reach some general aggrement will post the proposal for voting(the only thing is I can't post it on Snapshot so someone else will have too).

**Reply by Cindy884** [reply]
yes, you can post your proposal draft in GitHub to test first.

**Reply by Cindy884** [reply]
Hello Saso, would you like to tell me your idea about KTON SubDAO? If ok, your can share me with your draft.:)

**Reply by SasoLithops** [reply]
Hi Cindy!

I will start a discussion on this as well. Playing with some ideas nothing final yet.
